build/assets/tuned/openshift-node-performance

```diff
@@ -121,12 +129,24 @@ avc_cache_threshold=8192
 initrd_remove_dir=
 initrd_dst_img=
 initrd_add_dir=
+
 # overrides cpu-partitioning cmdline
-cmdline_cpu_part=+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} intel_pstate=disable nosoftlockup
+cmdline_cpu_part=+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} systemd.cpu_affinity=${not_isolated_cores_expanded} intel_iommu=on iommu=pt
+
 {{if .StaticIsolation}}
-cmdline_realtime=+tsc=nowatchdog intel_iommu=on iommu=pt isolcpus=domain,managed_irq,${isolated_cores} systemd.cpu_affinity=${not_isolated_cores_expanded}
+cmdline_isolation=+isolcpus=domain,managed_irq,${isolated_cores}
 {{else}}
-cmdline_realtime=+tsc=nowatchdog intel_iommu=on iommu=pt isolcpus=managed_irq,${isolated_cores} systemd.cpu_affinity=${not_isolated_cores_expanded}
+cmdline_isolation=+isolcpus=managed_irq,${isolated_cores}
+{{end}}
+
+{{if .RealTime}}
+cmdline_realtime=+tsc=nowatchdog nosoftlockup nmi_watchdog=0 audit=0 mce=off skew_tick=1
 {{end}}
+
+{{if not .PowerSaving}}
+cmdline_power_performance=+processor.max_cstate=1 intel_idle.max_cstate=0 idle=poll intel_pstate=disable
```

Comment:
Since you idle=poll is being set why would processor.max_cstate=1 intel_idle.max_cstate=0 need to be set ?